app/controllers/apartments_controller.rb

```diff
@@ -4,4 +4,10 @@ def index
         render json: apartments
     end
 
+    def update
```

Comment:
How are you handling the request in the controller if Apartment is not valid?

Reply:
Am I waiting for the 'create' to be here first?

Reply:
I'm confused here I think.

Reply:
No need to wait for any info, but you do want to handle the request if apartment is not able to be updated.  (See create in cat tinder, this will look like a conditional)